SimPEG/Utils/modelutils.py

```diff
@@ -29,32 +34,42 @@ def surface2ind_topo(mesh, topo, gridLoc='CC', method='nearest', fill_value=np.n
 
         if gridLoc == 'CC':
             XY = ndgrid(mesh.vectorCCx, mesh.vectorCCy)
-            Zcc = mesh.gridCC[:, 2].reshape((np.prod(mesh.vnC[:2]), mesh.nCz), order='F')
-            #gridTopo = Ftopo(XY)
-            gridTopo = griddata(topo[:, :2], topo[:, 2], XY, method=method, fill_value=fill_value)
-            actind = [gridTopo >= Zcc[:, ixy] for ixy in range(np.prod(mesh.vnC[2]))]
+            Zcc = mesh.gridCC[:, 2].reshape((np.prod(mesh.vnC[:2]), mesh.nCz),
+                                            order='F')
+
```

Comment:
This is me being a bit picky, but I find it much easier to read across multiple lines if we break at brackets, eg
```
Zcc = mesh.gridCC[:, 2].reshape(
    (np.prod(mesh.vnC[:2]), mesh.nCz), order='F'
)
gridTopo = griddata(
    topo[:, :2], topo[:, 2], XY, method=method, fill_value=fill_value
)
actind = [
    gridTopo >= Zcc[:, ixy] for ixy in range(np.prod(mesh.vnC[2]))
]
``` 

No worries if you don't want to update this on this pr, just a thought for the future. cc @rowanc1: might be worth putting a few notes on style in the docs. Thoughts on that? Do you have any handy references?